source/_posts/free-first-chapter-of-jumpstart-for-everyone.md

```diff
@@ -0,0 +1,32 @@
+---
+title: First chapter of Machine Learning Jump Start will be free
```

Comment:
Since it basically already happened when they read this, how about:

"First chapter of Machine Learning Jump Start freely available for everyone"

Reply:
Or something like this. Maybe @PascalPrecht has another nice title idea.